ext/libxml/tests/libxml_get_external_entity_loader.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+libxml_get_external_entity_loader() returns current handler
+--SKIPIF--
+<?php if (!extension_loaded('dom')) die('skip dom extension not available'); ?>
```

Comment:
you should use the `--EXTENSIONS--` section here as well

Reply:
Done. Weird, I guess I must have started this patch against some old branch.